rotkehlchen/api/v1/encoding.py

```diff
@@ -969,6 +986,58 @@ def make_ethereum_transaction_query(  # pylint: disable=no-self-use
         }
 
 
+class TradesQuerySchema(
+        AsyncQueryArgumentSchema,
+        OnlyCacheQuerySchema,
+        DBPaginationSchema,
+        DBOrderBySchema,
+):
+    base_asset = AssetField(load_default=None)
+    quote_asset = AssetField(load_default=None)
+    from_timestamp = TimestampField(load_default=Timestamp(0))
+    to_timestamp = TimestampField(load_default=ts_now)
+    trade_type = TradeTypeField(load_default=None)
+    location = LocationField(load_default=None)
+
+    @validates_schema
+    def validate_trades_query_schema(  # pylint: disable=no-self-use
+            self,
+            data: Dict[str, Any],
+            **_kwargs: Any,
+    ) -> None:
+        value = data['order_by_attribute']
+        if data['order_by_attribute'] not in (None, 'time'):
```

Comment:
shouldn't this has to be `timestamp`? @LefterisJP

Reply:
Nevermind, it's supposed to be `time` I think. I will convert it in frontend

Reply:
answered you in discord